lambdas/thumbnail/requirements.txt

```diff
@@ -1,10 +1,14 @@
+#aicsimageio==3.0.0
```

Comment:
Uncomment it?

Also, please include all dependencies - i.e., dependencies of `aicsimageio`, etc. - not just the direct ones. `pip freeze` is probably the easiest way - just make sure you don't have `pytest`-related stuff there, and remove the lambda itself.